### DISCUSSION TITLE
GitHub ARM64 runners are now available

### DISCUSSION THREAD

**Discussion by czyt** [rustdesk/rustdesk#8270] [General]
maybe rustdesk can build with github arm64 runner

**Comment by rustdesk** [comment]
https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta/#:~:text=We%20expect%20to%20begin%20offering%20Arm%20runners%20for%20open%20source%20and%20personal%20accounts%20by%20the%20end%20of%20the%20year.
